tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/Analyzer.scala

```diff
@@ -336,8 +336,9 @@ final class Analyzer(semantics: Semantics, outputMode: OutputMode,
     }
 
     def tryLookupMethod(methodName: String): Option[MethodInfo] = {
-      assert(isScalaClass,
+      assert(isScalaClass || isInterface,
           s"Cannot call lookupMethod($methodName) on non Scala class $this")
+
       @tailrec
       def loop(ancestorInfo: ClassInfo): Option[MethodInfo] = {
```

Comment:
This probably needs a better name now... How about `lookupInherited` or something like that?